mr/src/main/java/org/apache/mahout/cf/taste/impl/similarity/precompute/MultithreadedBatchItemSimilarities.java

```diff
@@ -26,7 +26,7 @@
 import java.util.concurrent.TimeUnit;
 import java.util.concurrent.atomic.AtomicInteger;
 
-import com.google.common.collect.Lists;
+import java.util.ArrayList;
 import com.google.common.io.Closeables;
```

Comment:
U should be able to get rid of Closeables too, and replace that with Java 7 try - resources;  don't worry about that I'll take care of it. JFyi.